packages/test-app-frontend/src/App/ITwinJsApp/AppUi/AppUiVisualizationHandler.ts

```diff
@@ -0,0 +1,156 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { UiFramework, WidgetState } from "@itwin/appui-react";
+import {
+  ChangedElementsListComponent, ChangedElementsWidget, type MainVisualizationOptions,
+  type SideBySideVisualizationManager, type VersionCompareManager, type VersionCompareVisualizationManager,
+  type VisualizationHandler
+} from "@itwin/changed-elements-react";
+
+import { PropertyComparisonFrontstage } from "./PropertyComparisonFrontstage";
+import { VersionCompareFrontstageManager } from "./VersionCompareFrontstageManager";
+
+export interface AppUiVisualizationOptions {
+  /* Frontstage Ids where version compare will be available. */
+  frontstageIds: string[];
+}
+
+/**
+ * Handles setting up version compare visualization for AppUi applications. This expects that the app has frontstages
+ * and adds our own property compare frontstage.
+ */
+export class AppUiVisualizationHandler implements VisualizationHandler {
```

Comment:
Will you be adding a similar one in version-compare package so that it's not up to the app to define this?

Reply:
Sorry, I may not be understanding this question correctly.

`version-compare` will contain the same sort of code that has now been added to the test app.

Reply:
Ok that answers my question. Thanks.